pyproject.toml

```diff
@@ -23,52 +24,39 @@ classifiers = [
     "Operating System :: OS Independent",
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
+dependencies = [
+    "arrow >=1.3.0",
```

Comment:
I thought about making this an extra (e.g. `dictdeeper[arrow]`), but decided that should be in a different PR if we want to do that.

Arrow is a small dependency and we use it to normalize `datetime`, so I'm opting to keep things straightforward so we don't have to change stuff in `dictdeeper.matcher`.